internal/vrf/ips.go

```diff
@@ -0,0 +1,65 @@
+package vrf
+
+import (
+	"context"
+	"fmt"
+
+	metal "github.com/equinix/equinix-sdk-go/services/metalv1"
+	"github.com/spf13/cobra"
+)
+
+func (c *Client) ips() *cobra.Command {
+	var (
+		vrfID string
+		ipID  string
+	)
+
+	// createVRFCmd represents the creatVRF command
+	ipsVRFCmd := &cobra.Command{
+		Use:   "ips [-i <VrfID] [-p <ip_id>]",
+		Short: "Retrieves the list of VRF IP Reservations for the VRF.",
+		Long:  "Retrieves the list of VRF IP Reservations for the VRF.",
+		Example: ` # Retrieves the list of VRF IP Reservations for the VRF..
+	
+	metal vrf ips [-i <VrfID] 
+
+	# Retrieve a specific IP Reservation for a VRF
+	metal vrf ips [-i <VrfID] [-p <ip_id>]`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SilenceUsage = true
+			var (
+				// vrfIpReservationList *metal.VrfIpReservationList
+				vrfIpReservation *metal.VrfIpReservation
+				err              error
+			)
+			inc := []string{}
+			exc := []string{}
+			ipIdFlag, _ := cmd.Flags().GetString("ipID")
+			if ipIdFlag == "" {
+				_, _, err = c.Service.FindVrfIpReservations(context.Background(), vrfID).Include(c.Servicer.Includes(inc)).Exclude(c.Servicer.Excludes(exc)).Execute()
+			} else {
+				vrfIpReservation, _, err = c.Service.FindVrfIpReservation(context.Background(), vrfID, ipID).Include(c.Servicer.Includes(inc)).Exclude(c.Servicer.Excludes(exc)).Execute()
+			}
+			if err != nil {
+				return fmt.Errorf("could not Find VrfIpReservation: %w", err)
+			}
+
+			data := make([][]string, 1)
+
+			// data[0] = []string{vrfRequest.GetName(), vrfRequest.GetDescription(), strconv.Itoa(int(vrfRequest.GetLocalAsn())), strings.Join(vrfRequest.GetIpRanges(), ","), vrfRequest.GetCreatedAt().String()}
+			// header := []string{"ID", "Name", "Description", "LocalASN", "IPranges", "Created"}
+			data[0] = []string{}
+			header := []string{}
+
+			return c.Out.Output(vrfIpReservation, header, &data)
+		},
+	}
+
+	ipsVRFCmd.Flags().StringVarP(&vrfID, "vrfID", "v", "", "VRF UUID")
```

Comment:
Our naming convention for an ID argument would have this field named: `vrf-id`. IP would be `--id` in this case, it is the ID of the named subcommand (IP).

The description should be more verbose; perhaps since these arguments change the behavior of the command:
- `Specify the VRF UUID to list its associated IP reservations.`
- `Specify the IP UUID to retrieve the details of a VRF IP reservation.`

Reply:
marking this conversation as resolved because this thread became disconnected from the relevant code, which still needs to be updated based on the original review.